### PR TITLE
Added an admin UI

### PR DESCRIPTION
**Pull request by tingaloo**
* UI addition does not break current code nor modify existing styles.
* Admin dashboard follows standard dashboard layout.
* Color scheme can be reworked to match CodeForDC, not sure how to implement red.
* Hopefully this will inspire some ideas.

image

### UI Suggestions:

1. Add persistent breadcrumbs, it doesn't make sense for a user to scroll to the very top to remember who the current client is. I was thinking of a jQuery drop-bar.

2. Pages need to be <b>modularized</b> for easier accessibility. But I think that requires heavily modifying controller logic.

What does everyone think? I was planning on writing some tests before modifying the controllers for increased modularity.

### CODE REVIEW

**Comment by tingaloo** [comment]
Slight bug: Anchors jumps will cut off the section heading.

image

File: app/views/application/_breadcrumb.html.erb [hunk 1]
```diff
@@ -6,10 +6,10 @@
       <span class="active">
       <% if defined? @model.my_number %>
         <%=@model.my_number%> -
-      <% end %>      
+      <% end %>
```
Comment (jrunningen) [inline app/views/application/_breadcrumb.html.erb]: Trimming whitespace!  Fantastic.

**Comment by jrunningen** [comment]
Looks cool.  I'll merge it in so the rest of the team can play around with the UI on the demo site.

**Comment by mlouie** [comment]
@tingaloo I love this.  I really dig that you isolated the two big actions we're expecting case workers to make and then separated out the other menu items on the left.  Looks clean and more intuitive to me.

**Pull request merged**